save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectDemoMenu.kt

```diff
@@ -72,21 +74,23 @@ val projectDemoMenu: FC<ProjectDemoMenuProps> = FC { props ->
         }
     }
 
-    useRequest {
-        val infoResponse = get(
-            "$apiUrl/demo/${props.organizationName}/${props.projectName}",
-            jsonHeaders,
-            ::loadingHandler,
-            ::noopResponseHandler,
-        )
-        if (infoResponse.ok) {
-            val demoInfo: DemoInfo = infoResponse.decodeFromJsonString()
-            setDemoStatus(demoInfo.demoStatus)
-            setDemoToolRequest(demoInfo.demoDto)
-            setGithubProjectCoordinates(demoInfo.demoDto.githubProjectCoordinates.orEmpty())
-        } else {
-            props.updateErrorMessage(infoResponse.statusText, infoResponse.unpackMessage())
-            setDemoStatus(DemoStatus.NOT_CREATED)
+    props.projectRole.isHigherOrEqualThan(Role.ADMIN).let {
```

Comment:
`isHigherOrEqualThan` returns `Boolean`, `let` is run for `true` and `false`. I guess you need to call here `runIf`

Reply:
I guess organization owner should be able to do it

Reply:
Added